test/gst-va/encode/vp9.py

```diff
@@ -0,0 +1,172 @@
+###
+### Copyright (C) 2022-2023 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.gstreamer.va.util import *
+from ....lib.gstreamer.va.encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "8bit")
+
+@slash.requires(*have_gst_element("vavp9lpenc"))
+@slash.requires(*have_gst_element("vavp9dec"))
+class VP9EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec         = "vp9",
+      gstencoder    = "vavp9lpenc",
```

Comment:
This should not be set here.  This base class should not know if it's lp vs. non-lp.  The distinction needs to be made in next subclasses.